docs/Developers/Useful_Links.rst

```diff
@@ -44,3 +44,11 @@ coala-Links
 -   `coala Chat <https://gitter.im/coala/coala>`_
 -   :doc:`coala Newcomers' Guide<Newcomers_Guide>`
 -   :doc:`coala Review Process<Review>`
+
+Regex-Links
+-----------
+
+-   `Regex Tutorial/Overview <https://ryanstutorials.net/regular-expressions-tutorial/>`_
```

Comment:
--> `Regex Overview/Tutorial`
Overview is more general than a tutorial, I think from reading-flow perspective it should come first :+1: